CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed etcd pod member names to be unique by having a random suffix instead of a sequence number e.g `example-etcd-cluster-2885z`
```

Comment:
Compare what it looked like before and looks now.
Also mention about compatibility.